pyOpenSci/python-package-guide
# docs: change update translation sessions in nox

## What was done

Following the discussion in PR #397, this PR makes the following changes:
- Modify the `update-translations` session to only update translation files for languages in `RELEASE_LANGUAGES`.
- Added a new nox session `update-langugage` that can be used to update the translation files for a specific language.

This PR also fixes a small syntax issue in the `documentation.po` file for the Spanish translation (missing quote).

## How to test

Run `nox -s update-translations` and confirm in the output that no languages are updated
```
❯ nox -s update-translations
nox > Running session update-translations
nox > Re-using existing virtual environment at .nox/update-translations.
nox > No release languages defined in RELEASE_LANGUAGES
nox > Session update-translations was successful.
```

Run `nox -s update-language -- es` and verify that the .po files for the Spanish translation are updated
```
❯ nox -s update-language -- es
(... some lines removed ...)
The message catalogs are in _build/gettext.
nox > Updating .po files for [es] translation
nox > sphinx-intl update -p _build/gettext -l es
Update: locales/es/LC_MESSAGES/tests.po +1, -1
Update: locales/es/LC_MESSAGES/documentation.po +1, -1
Update: locales/es/LC_MESSAGES/index.po +2, -2
Update: locales/es/LC_MESSAGES/tutorials.po +41, -19
Update: locales/es/LC_MESSAGES/TRANSLATING.po +2, -2
Update: locales/es/LC_MESSAGES/CONTRIBUTING.po +68, -15
Update: locales/es/LC_MESSAGES/package-structure-code.po +18, -18
nox > Session update-language was successful.
```
